uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -97,7 +97,7 @@ def __init__(
 
         # Connection events
         self.scope: WebSocketScope = None  # type: ignore[assignment]
-        self.handshake_started_event = asyncio.Event()
+        self.handshake_completed_event = asyncio.Event()
         self.handshake_completed_event = asyncio.Event()
```

Comment:
Eh... :thinking:

Reply:
I see!  I didn't notice this one.  Let me take another look at this.  Silly me.